fuzzers/fuzzbench_qemu/Makefile.toml

```diff
@@ -79,7 +79,7 @@ windows_alias = "unsupported"
 script_runner = "@shell"
 script='''
 rm -rf libafl_unix_shmem_server || true
-timeout 10s cargo run --release -- --libafl-in ../libfuzzer_libpng/corpus --libafl-out ./out ./${FUZZER_NAME} >fuzz_stdout.log || true
+timeout 30s bash -c 'cargo run --release -- --libafl-in ../libfuzzer_libpng/corpus --libafl-out ./out ./${FUZZER_NAME}' > fuzz_stdout.log || true
```

Comment:
I found qemu fuzzers doesn't work on GitHub runners. (perhaps OOM?)
the fuzzers never run.
thus I removed the tests in #1179 

so can you revert this change on this file?

Reply:
done